lab-examples/rtr/000/rtr000.clab.yml

```diff
@@ -0,0 +1,1715 @@
+
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark" data-a11y-animated-images="system">
+  <head>
+    <meta charset="utf-8">
+  <link rel="dns-prefetch" href="https://github.githubassets.com">
+  <link rel="dns-prefetch" href="https://avatars.githubusercontent.com">
```

Comment:
@frederic-loui this looks like some accidentally committed file

Reply:
commit `00a5e3a` should do the trick